docs/application/dotnet/guides/nui/window.md

```diff
@@ -313,6 +313,33 @@ child.Unparent();
 > If parent's window stack is changed by calling `Raise()` or `Lower()`, the child window is located on top of the parent again.
 >
 
+### Window Type
+
+NUI Window has serveral window types.
+The detail descrition is written in NUI Constants. See [WindowType] (application/dotnet/api/TizenFX/latest/api/Tizen.NUI.WindowType.html).
+In addition, we can set window type in NUIApplicatiion's constructor.
```

Comment:
Is this a mandatory or an optional step?
```suggestion
You can also set the window type in the NUIApplication's constructor.
```

Reply:
It is optional.
But, if  application wants to use IME window, this constructor should be used.